mlflow/pyfunc/backend.py

```diff
@@ -27,13 +28,27 @@ class PyFuncBackend(FlavorBackend):
     """
 
     def __init__(
-        self, config, workers=1, env_manager=_EnvManager.CONDA, install_mlflow=False, **kwargs
+        self,
+        config,
+        workers=1,
+        env_manager=_EnvManager.CONDA,
+        install_mlflow=False,
+        env_root_dir=None,
+        **kwargs,
     ):
+        """
+        :param env_root_dir: Root path for conda env. If None, use Conda's default environments
+                             directory. Note if this is set, conda package cache path becomes
+                             "env_root_dir/conda_cache_pkgs" instead of the global package cache
+                             path, and pip package cache path becomes "env_root_dir/pip_cache_pkgs"
```

Comment:
```suggestion
                             path, and pip package cache path becomes "{env_root_dir}/pip_cache_pkgs"
```

same here https://github.com/mlflow/mlflow/pull/5561/files#r845689485